compiler/parser/parser.go

```diff
@@ -231,3 +231,17 @@ func (p *Parser) IsNotDefMethodToken() bool {
 
 	return p.curToken.Type != token.Ident && !(p.peekToken.Type == token.Dot && (p.curToken.Type == token.InstanceVariable || p.curToken.Type == token.Constant || p.curToken.Type == token.Self))
 }
+
+// Token type InstanceVariable and Constant will trigger IsNotParamsToken()
+var invalidParams = map[token.Type]bool{
```

Comment:
I think the return value bool is useless but I don't know whether there is any better way to implement this mapping table

Reply:
I think this is the only way for that